docker-compose.override.cassandra-dse.yml

```diff
@@ -0,0 +1,92 @@
+volumes:
+  common-cassandra-db-dse:
+    name: "common-cassandra-db-dse"
+
+services:
+  cassandra-node1:
+    image: datastax/dse-server:6.8.16-ubi7
+    container_name: cassandra-node1
+    hostname: cassandra-node1
+    networks:
+      common-net:
+        aliases:
+          - common-cassandra-node1
+    # Maps cassandra exercises to a local folder.
+    # This preserves data across container restarts.
+    volumes:
+      - ./.volume-backups/tempdb:/opt/dse/tempdb
+      - ./cassandra-dse/dse.yaml:/opt/dse/resources/dse/conf/dse.yaml
+    env_file:
+      - .envs/cassandra-dse.env
+    # Exposing ports for inter cluster communication
+    expose:
+      # Intra-node communication
+      - 7000
+      # TLS intra-node communication
+      - 7001
+      # JMX
+      - 7199
+      # CQL
+      - 9042
+      # CQL SSL
+      - 9142
+    ports:
+      - 9042:9042
+    ulimits:
+      memlock: -1
+      nproc: 32768
+      nofile: 100000
+  cassandra-node2:
+    image: datastax/dse-server:6.8.16-ubi7
+    container_name: cassandra-node2
+    hostname: cassandra-node2
+    networks:
+      common-net:
+        aliases:
+          - common-cassandra-node2
+    volumes:
+      - ./.volume-backups/tempdb:/opt/dse/tempdb
+      - ./cassandra-dse/dse.yaml:/opt/dse/resources/dse/conf/dse.yaml
+    env_file:
+      - .envs/cassandra-dse.env
+    expose:
+      - 7000
+      - 7001
+      - 7199
+      - 9042
+      - 9142
+    ports:
+      - 9043:9042
```

Comment:
Ports section should be moved to docker-compose.override.example.yml file.

Reply:
The ports section is moved to docker-compose.override.example.yml file also the host ports are defined in the .env.example